src/components/rangeslider/draw.js

```diff
@@ -352,7 +353,8 @@ function drawRangePlot(rangeSlider, gd, axisOpts, opts) {
 
         mockFigure.layout[oppAxisName] = {
             domain: [0, 1],
-            range: oppAxisOpts.range.slice()
+            range: oppAxisOpts.range.slice(),
+            calendar: oppAxisOpts.calendar
```

Comment:
@etpinard apologies for adding a nearly duplicate mock to test this... but since the fix is in `draw` I didn't really see another way. I tried to at least put both rangesliders together on one plot but it didn't seem to like that at all.